docs/contributing/style-guide.md

```diff
@@ -1168,9 +1201,25 @@ Never use Sumo Logic in the plural or possessive form.
 
 Punctuation is placed outside of quotation marks, British English style. For everything else (dates, times, spelling), we use American English style.
 
+### Colons
+Colons are used to introduce lists or to separate titles from subtitles. Only include the colon if the introduction isn’t a complete sentence.
+
 ### Commas
 
-We use the Oxford (serial) comma. For example, use "I had eggs, toast, and orange juice", not "I had [eggs, toast and orange juice](https://www.verbicidemagazine.com/wp-content/uploads/2012/01/why-i-still-use-the-oxford-comma.jpg)".
+We use the Oxford (serial) comma. For example, use "I had eggs, toast, and orange juice", not "[I had eggs, toast and orange juice](https://www.verbicidemagazine.com/wp-content/uploads/2012/01/why-i-still-use-the-oxford-comma.jpg)".
+
+### Exclamation Points
+
+Use exclamation points to express excitement or encourage the user. Don't use them for errors, warnings, or confirmation of basic actions as they are usually unnecessary and can distract from important details.
+
+### Ellipses and Truncation
+
+Use ellipses for truncation. Don’t use ellipses for placeholders or trailing off a sentence unless it is getting cut off. When truncating, think about which part of the string is most essential for the user. This may require truncating in the beginning, middle, or end of a string.
+
+### Periods
+
+In the UI, avoid periods for single sentences on their own. Whenever there are two or more sentences, use periods at the end of each sentence. Separate sentences by one space, not two.
+
 
 
 ## Release Notes
```

Comment:
```suggestion
## Release notes
```

Reply:
You should also change "Reusing Content" to "Reusing content".

Reply:
thank you - done@